esmvaltool/cmorizers/obs/utilities.py

```diff
@@ -492,7 +492,7 @@ def roll_cube_data(cube, shift, axis):
 
 def set_units(cube, units):
     """Set units in compliance with cf_unit."""
-    special = {'psu': 1.e-3, 'Sv': '1e6 m3 s-1'}
+    special = {'psu': 1, 'Sv': '1e6 m3 s-1'}
```

Comment:
Why is that needed (asking as a non-specialist)?

Reply:
this is to comply with `psu` units definition of cmor vocabulary.